frontend/src/manage.tsx

```diff
@@ -58,12 +60,15 @@ export function Manage() {
   const [target, setTarget] = useState(undefined as string | undefined)
   const [options, setOptions] = useState([] as any[])
   const [selected, setSelected] = useState(undefined as any)
+  const [showInactive, setShowInactive] = useState(false)
 
   /**
    * Set the options of the main select box to the list of current users.
    */
   const setUserOptions = () => {
-    const userOptions: NameID[] = users.map((u: NameID) => {return {value: u.id, label: userLabel(u)}});
+    const userOptions: OptionType[] = users.map((u: NameID) => {
+      return {value: `u.id`, label: userLabel(u)}
```

Comment:
```suggestion
      return {value: u.id, label: userLabel(u)}
```

Reply:
yeah - it needs to be `'' + u.id` (or similar cast to string), I noticed that too ^^